sled-agent/src/http_entrypoints.rs

```diff
@@ -484,15 +507,15 @@ impl SledAgentApi for SledAgentImpl {
 
     async fn sled_role_get(
         rqctx: RequestContext<Self::Context>,
-    ) -> Result<HttpResponseOk<SledRole>, HttpError> {
+    ) -> Result<HttpResponseOk<v1::inventory::SledRole>, HttpError> {
```

Comment:
This should probably be an import from `sled-agent-types` right?

```suggestion
    ) -> Result<HttpResponseOk<SledRole>, HttpError> {
```

Reply:
No because this is a removed endpoint, for which we use versioned identifiers.